src/pages/create.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Button,
+  Flex,
+  FormControl,
+  Heading,
+  Input,
+  Stack,
+  Link
+} from '@chakra-ui/react';
+import { Children } from 'react';
+
+
+export default function Create() {
+  
+  const handleClick = () => {
+    <Button
+      isLoading
```

Comment:
@kevinjung13 This is not how you should do it
You must use a state, `useState` 
`const [isLoading, setLoading] = useState(false)`
And then when you handle the click, you update the state 

Your button below get the prop isLoading={isLoading}